sortedm2m_tests/test_base.py

```diff
@@ -352,20 +351,4 @@ def set_up_test_model(
                     bases=["%s.Pony" % app_label],
                 )
             )
-        if manager_model:
-            from .models import FoodManager, FoodQuerySet
```

Comment:
`FoodManager` does not exist in that module. These were adapted from Django, which explains that. I can't imagine this library needing to test this.